src/controllers/user.controller.js

```diff
@@ -131,4 +131,21 @@ export default class UserController {
       Response.UnauthorizedError(res, { message: 'Unable to sign in' });
     }
   }
+
+  /**
+ * get all managers
+ * @param {ServerRequest} req
+ * @param {ServerResponse} res
+ * @returns {ServerResponse} response
+ */
+  static async getManagers(req, res) {
+    try {
+      const result = await User.findAll({ attributes: ['id', 'first_name', 'last_name', 'email', 'preferred_language', 'location'], where: { role: 'manager' } });
+      return Response.Success(res, {
+        ...result
```

Comment:
`result` is an array, why don't you pass it is ```Response.Success(res, result)``` rather than as an object?

Reply:
@Qausim, i didnt understand it that way,  i think this is the method i have used thus far in this project while working with findAll

Reply:
Please return the result as an array.